lib/state/ui/reducer.ts

```diff
@@ -11,6 +11,19 @@ const filteredNotes: A.Reducer<T.NoteEntity[]> = (
   action
 ) => ('FILTER_NOTES' === action.type ? action.notes : state);
 
+const listTitle: A.Reducer<string> = (state = 'All Notes', action) => {
+  switch (action.type) {
+    case 'App.showAllNotes':
+      return 'All Notes';
+    case 'App.selectTrash':
+      return 'Trash';
+    case 'App.selectTag':
+      return action?.tag?.data?.name;
```

Comment:
Can this be `undefined`/`null`? I did some searching and couldn't find anywhere that it's called without a `T.TagEntity`

Reply:
I guess now that I have tools I just started using them.  You are right I went too far here.

Reply:
Done